ignf_gpf_api/_conf/json_schemas/workflow.json

```
@@ -39,7 +39,8 @@
                                             "enum": [
                                                 "processing-execution",
                                                 "configuration",
-                                                "offering"
+                                                "offering",
+                                                "datastore"
```

Ici c'est les différents types d'actions, donc c'est pas ici qu'il faut ajouter datastore, mais bien ligne 61.
```suggestion
                                                "offering"
```